app/signals/apps/my_signals/models.py

```diff
@@ -1,30 +1,30 @@
 # SPDX-License-Identifier: MPL-2.0
-# Copyright (C) 2022 Gemeente Amsterdam
-import binascii
-import os
+# Copyright (C) 2022 - 2023 Gemeente Amsterdam
 from datetime import timedelta
 
 from django.db import models
 from django.utils.timezone import now
 
 from signals.apps.my_signals.app_settings import MY_SIGNALS_TOKEN_EXPIRES_SECOND
+from signals.apps.signals.tokens.token_generator import TokenGenerator
 
 
 class Token(models.Model):
     """
     Based on the AuthToken implementation of rest-framework
     """
-    key = models.CharField(max_length=40, primary_key=True)
+    key = models.CharField(max_length=120, primary_key=True)
```

Comment:
Why change this to 120 instead of providing an `nbytes` so that the `key` still fits within the 40 characters? I see that it is currently rendering tokens with a length of 43.

Reply:
https://docs.python.org/3/library/secrets.html#secrets.token_urlsafe

Currently by default 32 bytes are used which is considered a reasonable default, as in it should be hard enough to crack right now. However over time this value will change (become larger) to compensate for systems becoming more powerful and thus requiring a more difficult to crack token.
When using 32 bytes and converting it to hex notation and then base64 encoding it (which is what `token_urlsafe()` does, 40 characters is not always enough to be able to store  the token.